R/regularized_regression.R

```diff
@@ -436,15 +439,14 @@ lasso_weights <- function(X, y) glmnet_weights(X, y, 1)
 
 #' @examples
 #' wgt.mr.ash <- mrash_weights(eqtl$X, eqtl$y_res, beta.init = lasso_weights(X, y))
-#' @importFrom mr.ash.alpha mr.ash
 #' @importFrom stats predict
 #' @export
 mrash_weights <- function(X, y, init_prior_sd = TRUE, ...) {
   args_list <- list(...)
   if (!"beta.init" %in% names(args_list)) {
     args_list$beta.init <- lasso_weights(X, y)
   }
-  fit.mr.ash <- do.call("mr.ash", c(list(X = X, y = y, sa2 = if (init_prior_sd) init_prior_sd(X, y)^2 else NULL), args_list))
+  fit.mr.ash <- do.call("mr.ash.alpha::mr.ash", c(list(X = X, y = y, sa2 = if (init_prior_sd) init_prior_sd(X, y)^2 else NULL), args_list))
```

Comment:
we need to check for existance of mr.ash.alpha at the start of this function

Reply:
Thanks, I missed this one!